qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -29,9 +29,30 @@ fi
 . $(dirname $0)/rbd_mirror_helpers.sh
 setup
 
-testlog "TEST: add image and test replay"
+testlog " - mirror default namespace with remote-namespace"
 start_mirrors ${CLUSTER1}
 image=test
+NS0=ns0
+rbd --cluster ${CLUSTER1} mirror pool enable ${POOL}
+rbd --cluster ${CLUSTER2} mirror pool enable ${POOL}
+rbd --cluster ${CLUSTER1} namespace create ${POOL}/${NS0}
+rbd --cluster ${CLUSTER2} mirror pool enable ${POOL} ${MIRROR_POOL_MODE} --remote-namespace ${NS0}
+rbd --cluster ${CLUSTER1} mirror pool enable ${POOL}/${NS0} ${MIRROR_POOL_MODE} --remote-namespace ""
```

Comment:
@nbalacha `--remote-namespace ""` here default namespace of which pool it considers?

Reply:
`default to default` namespace is just RBD image based mirroring Is it?

Reply:
> `--remote-namespace ""` here default namespace of which pool it considers?

Of the pool with the same name in the remote cluster.  Remapping between pools isn't supported -- only remapping between namespaces within the same pool.

> default to default namespace is just RBD image based mirroring Is it?

Not necessarily image-based, it can be pool-based is well.  Default to default namespace mirroring is RBD mirroring as it existed for years, the addition of namespace remapping support doesn't introduce any constraints on the mirror mode.